### ISSUE TITLE
Since 20.3 mintupdate can only be launched once via tray icon after system start (in MATE using Compiz)

### ISSUE THREAD

**Issue by Alyana2714** [linuxmint/mintupdate#722]
**Describe the bug**
After starting the system the tray Icon of mintupdate has limited functionality. It can be left clicked, but only launches mintupdate the first time you do that.
Clicking it with left mouse button again afterwards has no effect anymore. It can be right clicked and all the options do work, but the main screen of mintupdate itself does not show up anymore.
This happened after upgrading Mint MATE 20.2 to 20.3, and also on fresh installations of Mint MATE 20.3.
It only happens when using Compiz as the display manager.
This did not happen when using Compiz in MATE 20.2.

**To Reproduce**
1. Install Mint MATE 20.3 or upgrade to 20.3 from 20.2
2. Use Compiz as the display manager
3. Restart system
4. Left click on the mintupdate tray icon
5. mintupdate will launch normally again. Either install updates or not, it does not matter
6. Close mintupdate again
7. Click on the mintupdate tray icon again with the left mouse button. The main screen of mintupdate itself does not show up anymore from this moment on. It can be right clicked, and the options can be launched, but not the main screen anymore.
8. The only way to launch mintupdate until the next restart is launching it from the main menu.
9. Restarting the system will enable the tray icon again for a single launch of mintupdate. Repeat from step 4.

**Expected behavior**
Mintupdate should launch, like it did in 20.2, when left clicking the tray icon. It is confusing for new users or beginners if they see the tray icon showing new updates and clicking on it has no effect.

**Distribution:**
- [x ] Linux Mint 20.3 MATE with Compiz
- [ ] LMDE
- [ ] Other (please specify)

**Software version:**
5.8.1

**Logs:**
02.23@09:45 ++ Launching Update Manager
02.23@09:45 ++ Changes to the package cache detected, triggering refresh
02.23@09:45 ++ Starting refresh (local only)
02.23@09:45 ++ Initial refresh will happen in 0 day(s), 0 hour(s) and 10 minute(s)
02.23@09:45 ++ System is up to date
02.23@09:45 ++ Refresh finished

**Crash report:**
no crashes. It just does not open from tray.

**Additional context**
It worked fine in 20.2 MATE, also using Compiz, so there must be something in 20.3 that changed this. I also tried to hide tray icon until new updates would be available, but this did not help either.

**Comment by wraith3690** [comment]
I have this bug as well on both of my systems only one click on tray icon when using compiz then it does not work again until restart however using command "mintupdate" in the terminal does open the window 

terminal output

`chris@chris-System-Product-Name:~$ mintupdate 
chris@chris-System-Product-Name:~$ /usr/lib/linuxmint/mintUpdate/mintUpdate.py:1285: DeprecationWarning: Gdk.threads_init is deprecated
  Gdk.threads_init()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1433: DeprecationWarning: Gtk.ImageMenuItem.set_image is deprecated
  menuItem3.set_image(image)
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:1670: DeprecationWarning: Gdk.threads_enter is deprecated
  Gdk.threads_enter()
/usr/lib/linuxmint/mintUpdate/mintUpdate.py:740: DeprecationWarning: Gdk.threads_leave is deprecated
  Gdk.threads_leave()
`

behaviour and terminal output is the same on both systems both where updated from mate 20.2     

system info PC 1
CPU  i5 8600k
GPU gtx 1070 driver 510.54
kernel 5.13.0-30-generic
software version 5.8.1

system info PC 2
CPU i5 9600k
GPU gtx 1070 driver 510.54
kernel 5.13.0-30-generic 
software version 5.8.1

**Comment by Alyana2714** [comment]
Yes, same here. Only the tray icon itself is affected by this bug. Any other way to launch mintupdate works (Terminal, Mint Menu), and no crash then either.

Would be nice if the devs could have a look, since it is very confusing for new users who are used to just click on the tray icon.

**Comment by jolly-rosso** [comment]
Same issue here! The try icon run only one time after reboot. To use mintupdate after first time, you need to launch it by terminal. This happen on my system upgraded from mint 17 -> 18 ....to 20.3 but also on mint 20.3 fresh installed on others pc, virtual too.

**Comment by peneus** [comment]
I am having the same problem on mint 21 as well

**Comment by wraith3690** [comment]
I have updated to mint 21.1 fresh install on both systems and both still have this bug and the terminal output on both is the same as my last post
